client/sites/marketing/tools/style.scss

```diff
@@ -89,34 +91,40 @@
 
 .tools__feature-list {
 	position: relative;
-	left: calc(-0.5em);
+	left: calc(-0.5rem);
 	display: flex;
 	flex-wrap: wrap;
 	// margins of the items will "hang off the side" invisibly
-	width: calc(100% + 1em);
+	width: calc(100% + 1rem);
+
+	a.components-button.is-link {
+		text-decoration: none;
+	}
```

Comment:
I first wanted to suggest adding a font weight here to adhere to the designs. When I did that, I noticed that only some of the buttons received the higher font weight:
![CleanShot 2024-11-11 at 13 44 42@2x](https://github.com/user-attachments/assets/cc50aad8-8c2e-4444-962e-446de60d29d8)

Turns out some of the buttons here are rendered with the `<button>` element, while others with the `<a>` element. To accommodate both, I think we should update the selector here to only target the class name:

```suggestion
	.components-button.is-link {
		text-decoration: none;
		font-weight: 500;
	}
```

Reply:
Well spotted! I will fix those. I will make sure the CSS specificity applies correctly here

Reply:
Added here [d0994d5](https://github.com/Automattic/wp-calypso/pull/96232/commits/d0994d57f7a2d1988d1828189648769f9a45bdb2), the selector `a.components-button.is-link` was actually redundant as you suggested